conf/application.conf

```diff
@@ -9,6 +9,9 @@ play.cache.redis.timeout=20s
 
 consignmentapi.url="http://localhost:8080/graphql"
 
+akka.actor.allow-java-serialization="on"
+akka.actor.warn-about-java-serializer-usage="off"
```

Comment:
Where are we using Java serialization?

Reply:
The library which connects the play cache to redis needs it.

Reply:
I've added a comment to the first application.conf file